tests/security/grub_auth/grub_authorization.pm

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: grub2 supports restricting access to boot menu entries when building their
+#          images/appliances,so that only specified users can boot selected menu entries.
```

Comment:
Please wrap above 2 lines in 80 characters.

Reply:
sure